### PR TITLE
fix: remove hard-coded server name from test

### PR DESCRIPTION
**Pull request by brownoxford**
Replace hard-coded server name in DID resolution request with environment variable.

FIX: 113

### CODE REVIEW

**Comment by brownoxford** [comment]
Note that postman export caused quite a lot of whitespace changes here. This PR is best viewed while ignoring whitespace, which can be done by clicking the cog on the "files changed" tab and checking the "ignore whitespace" box.

File: docs/tutorials/revocation/revocation.collection.json [hunk 1]
```diff
@@ -1,285 +1,313 @@
...
+				"method": "GET",
+				"header": [
+					{
+						"key": "Accept",
+						"value": "application/json",
+						"type": "text"
+					}
+				],
+				"url": {
+					"raw": "{{API_BASE_URL}}/identifiers/{{ORGANIZATION_DID_WEB}}",
```
Comment (mkhraisha) [inline docs/tutorials/revocation/revocation.collection.json:73]: This looks like it includes the identifiers change too?

Comment (brownoxford) [reply docs/tutorials/revocation/revocation.collection.json:73]: ah - good catch. I'll pull that for now, as it should be part of a different PR that updates this URL in all postman tests.

File: docs/tutorials/revocation/revocation.collection.json [hunk 2]
```diff
@@ -70,12 +70,11 @@
 					}
 				],
 				"url": {
-					"raw": "{{API_BASE_URL}}/identifiers/{{ORGANIZATION_DID_WEB}}",
```
Comment (OR13) [inline docs/tutorials/revocation/revocation.collection.json]: I think we want this... the change should go the other direction... fixing all "resolution endpoints" to look like 

```
"{{API_BASE_URL}}/identifiers/{{ORGANIZATION_DID_WEB}}",
```

**Review by OR13 — CHANGES_REQUESTED** [review]
We want this fix to got the other way.

File: docs/tutorials/revocation/revocation.collection.json [hunk 3]
```diff
@@ -1,285 +1,312 @@
...
+				"method": "GET",
+				"header": [
+					{
+						"key": "Accept",
+						"value": "application/json",
+						"type": "text"
+					}
+				],
+				"url": {
+					"raw": "{{API_BASE_URL}}/{{ORGANIZATION_DID_WEB}}",
```
Comment (OR13) [inline docs/tutorials/revocation/revocation.collection.json:73]:
```suggestion
					"raw": "{{API_BASE_URL}}/identifiers/{{ORGANIZATION_DID_WEB}}",
```

Comment (OR13) [reply docs/tutorials/revocation/revocation.collection.json:73]: See https://w3c-ccg.github.io/did-resolution/#example-0

File: docs/tutorials/revocation/revocation.collection.json (hunk 2 above)
Comment (brownoxford) [reply docs/tutorials/revocation/revocation.collection.json]: Agreed - my apologies for acting without thinking this through. I've reverted the change.

**Review by OR13 — APPROVED** [review]

**Review by mprorock — APPROVED** [review]
approved with note to fix conflicts

**Review by nissimsan — APPROVED** [review]

**Pull request merged**